cognite/powerops/resync/config/_main.py

```diff
@@ -230,6 +248,12 @@ def from_yamls(cls, config_dir_path: Path, cdf_project: str) -> ReSyncConfig:
             # For backwards compatibility
             configs["constants"]["cdf_project"] = cdf_project
 
+        # Hack to instantiate CogShopConfig from dict
```

Comment:
Open for other suggestions here... this is very ugly

Reply:
Nah, I think this is fine :)